nucypher/characters/lawful.py

```diff
@@ -71,23 +72,48 @@ class Alice(Character, PolicyAuthor):
     _default_crypto_powerups = [SigningPower, DecryptingPower, DelegatingPower]
 
     def __init__(self,
-                 is_me=True,
-                 federated_only=False,
+                 checksum_address: str = None,
+                 m: int = None,
```

Comment:
Great that we allow to specify default `m` and `n`. Should we not offer some values here?

Reply:
I've got the default values specified on `AliceConfiguration`, leaving character decoupled from config.